tests/PerformanceHarness/log_reader.py

```diff
@@ -39,7 +39,7 @@ class TpsTestConfig:
     numTrxGensUsed: int = 0
     targetTpsPerGenList: List[int] = field(default_factory=list)
     quiet: bool = False
-    printMissingTransactions: bool=False
+    printMissingTransactions: bool=True
```

Comment:
The reason this was not defaulted to `True` initially was that if the transaction generators initially fail to connect completely, then the log file is riddled with potentially tens of thousands of log messages for dropped transactions.